### ISSUE TITLE
Allow for using source country as a criteria in zia_firewall_filtering_rule 

### ISSUE THREAD

**Issue by chris-rockwell** [zscaler/terraform-provider-zia#372]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add attribute to `zia_firewall_filtering_rule` to allow for filtering on source country.

### New or Affected Resource(s)

- zia_firewall_filtering_rule

### Potential Terraform Configuration

```hcl
resource "zia_firewall_filtering_rule" "country_block" {
  action  = "BLOCK_DROP"
  default_rule = false
  enable_full_logging = false
  name = "Block From Country"
  order = 1
  predefined = false
  rank = 7
  state = "ENABLED"
  dest_countries = ["ABC"]
}
```

**Comment by willguibr** [comment]
@chris-rockwell 
Can you please elaborate on your request regarding the attribute "dest_countries"?
What exactly do you mean by: "allow for filtering on source country"?
Thank you

Zscaler DevRel.

**Comment by willguibr** [comment]
@chris-rockwell 
The attribute `source_countries` is now available on version v3.0.6.

Zscaler DevRel
